client/verta/verta/client.py

```diff
@@ -1370,6 +1370,41 @@ def get_dataset_version(self, id):
             raise ValueError("Dataset Version not found")
         return dataset_version
 
+    # holdover for backwards compatibility
+    def find_datasets(self,
+                      dataset_ids=None, name=None,
+                      tags=None,
+                      sort_key=None, ascending=False,
+                      workspace=None):
+        warnings.warn(
+            "this method is deprecated and will removed in an upcoming version;"
+            " consider using `client.datasets.find()` instead",
+            category=FutureWarning,
+        )
+        datasets = self.datasets
```

Comment:
This method is copied over from `0.15.9`; this line is the only functional difference, used to be
```
datasets = Datasets(self._conn, self._conf)
```